db/log_reader.cc

```diff
@@ -279,7 +291,25 @@ bool Reader::ReadRecord(Slice* record, std::string* scratch,
         }
         break;
       }
-
+      case kUserDefinedTimestampSizeType:
```

Comment:
It's a good point to reorder these. I will start doing these nit refactor in separate commit in the future so that it will not make it harder to see what lines are changed.